trillian/integration/hammer.go

```diff
@@ -489,13 +492,21 @@ func (s *hammerState) addChainInvalid(ctx context.Context) error {
 	return nil
 }
 
+// chooseAddType determines whether to add a new or pre-existing cert.
+func (s *hammerState) chooseCertToAdd() Choice {
+	if s.cfg.DuplicateChance > 0 && rand.Intn(s.cfg.DuplicateChance) == 0 {
+		// TODO(drysdale): restore LastCert as an option
+		return FirstCert
```

Comment:
Can `FirstCert` be nil when we get here?

Reply:
Don't think so - `FirstCert` is a `const`...

Reply:
OK I see it now, thought it might have been holding onto a generated cert.

Reply:
Ah, yes - it does do that elsewhere in pre-existing code.